src/handbook/customer/hubspot.md

```diff
@@ -33,14 +33,14 @@ The following is a description of each stage.
 
 ##### From MQL to SQL - Qualifying questions
 
-1. What is your business, and where does Node-RED fit in?
-1. What are you building with Node-RED, and how is it helping your business?
-1. What challenges you've run into operating Node-RED?
-1. Why now?
+
+- Is there a NR use case?
```

Comment:
I don't think this represents our discussion since yesterday, can we update the PR?